modules/samples/oauth2/playground2/src/main/org/wso2/sample/identity/oauth2/PollingClient.java

```diff
@@ -0,0 +1,38 @@
+package org.wso2.sample.identity.oauth2;
+
+/**
+ * Created by hasanthi on 3/24/15.
+ */
+
+import com.sun.jersey.api.client.Client;
+import com.sun.jersey.api.client.ClientResponse;
+import com.sun.jersey.api.client.WebResource;
+
+public class PollingClient {
+    /**
+     *
+     * @return Response true/false
+     */
+    public String getOPResponse() {
+        String opResponse = null;
+        try {
+            Client client = Client.create();
+            WebResource webResource = client
+                    .resource("https://localhost:9443/oauth2/session");
```

Comment:
Add the url as a constant in the class and refer here. Why hardcoding hostname as localhost and port as 9443 ? They should be configurable or deriving from existing config.